ts/helpers/filehandler.ts

```diff
@@ -0,0 +1,15 @@
+//handle all zip file actions here
+import { promises as fs } from 'fs';
+import * as xml2js from "xml2js-es6-promise";
+
+export default class FileHandler {
+    public static async parseSlideAttributes(fileName) {
+        const presentationSlide = await fs.readFile(fileName, 'utf8');
```

Comment:
again, variable could be named better, suggestion 
```suggestion
        const presentationXML= await fs.readFile(fileName, 'utf8');
```